content/applications/general/in_app_purchase.rst

```diff
@@ -1,59 +1,179 @@
 =====================
-In-App Purchase (IAP)
+In-app purchase (IAP)
 =====================
 
-In-App Purchases (IAP) gives access to additional services through Odoo. For instance, it allows you
-to send SMS Text Messages or to send Invoices by post directly from your database.
+In-app purchases (IAP) are optional services that enhance Odoo databases. Each service provides its
+own specific features and functionality.
 
-.. _iap/buying_credits:
+.. example::
+   The :guilabel:`SMS` service sends text messages to contacts directly from the database, and the
+   :guilabel:`Documents Digitization` service digitizes scanned or PDF vendor bills, expenses, and
+   resumes with optical character recognition (OCR) and artificial intelligence (AI).
 
-Buying Credits
-==============
+IAP services do **not** need to be configured or set up before use. Odoo users can simply click on
+the service in the app, to activate it. However, each service requires its own prepaid credits, and
+when they run out, users **must** :ref:`buy more <iap/buying_credits>` in order to keep using it.
 
-Each IAP Service relies on prepaid credits to work and has its own pricing. To consult your current
-balance or to recharge your account, go to :menuselection:`Settings --> Odoo IAP --> View my
-Services`.
+.. note::
+   Users with an Enterprise version of Odoo Online get free credits to test IAP features.
 
-.. image:: in_app_purchase/image1.png
+.. image:: in_app_purchase/iap.png
    :align: center
+   :alt: The IAP catalog with various services available on IAP.Odoo.com.
 
-.. tip::
-   If you are on Odoo Online and have the Enterprise version, you benefit from free credits to test our
-   IAP features.
+.. _in_app_purchase/portal:
 
-IAP accounts
+IAP services
 ============
 
-Credits to use IAP services are stored on IAP accounts, which are specific to each service.
-By default, IAP accounts are common to all companies, but can be restricted to specific
-ones. Activate the :ref:`developer mode <developer-mode>`, then go to :menuselection:`Technical
-Settings --> IAP Account`.
+IAP services are provided by Odoo as well as third-parties, and have a wide range of use cases.
 
-.. image:: in_app_purchase/image2.png
-   :align: center
+The following are IAP services provided by Odoo:
 
-.. tip::
-   An IAP account can be disabled by appending `+disabled` to its token.
-   Reverting this change will re-enable the account.
+- :guilabel:`Documents Digitization`: digitizes scanned or PDF vendor bills, expenses, and resumes
+  with OCR and AI.
+- :guilabel:`Lead Generation`: generates leads based on a set of criteria and converts web visitors
+  into quality leads and opportunities.
+- :guilabel:`Partner Autocomplete`: automatically populates contact records with corporate data.
+- :guilabel:`SMS`: sends SMS text messages to contacts directly from the database.
+- :guilabel:`Snailmail`: sends customer invoices and follow-up reports by post, worldwide.
```

Comment:
I know/understand why the entire list isn't going to be laid out, but I think it might be best to list these in the same order they appear on the Catalog website that's being linked. So, I'd recommend re-ordering these.